package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "2.3.1",
+    "@docusaurus/plugin-client-redirects": "^2.4.1",
```

Comment:
Do we still need this? I guess it doesn't hurt but we are not using it anymore, right?

Reply:
Nope. I'll take it out 👍.